templates/pages/index.html

```diff
@@ -5,9 +5,13 @@
 {% block content %}
 	<!-- Main jumbotron for a primary message -->
 	<div class="jumbotron">
-		<h1>TrackCat</h1>
+        <div class="h1"> 
```

Comment:
I would also vote to more descriptive classes than `h1` and `h2`. In fact I am not a great fan of defining class names which are same as tags. It happens oh so quickly that one defines `<h1 class="h2"></h1>, which makes as much sense as "Iggily biggily"...

In your case than it would be much better to do
```
<div class="jumbotrone">
    <h1>TrackCat</h1>
</div>
```
and CSS
```
.jumbotrone h1{
    put the styles here
}